pages/rooms/index.tsx

```diff
@@ -115,28 +126,28 @@ export default function Rooms({
         <link rel='icon' href='/favicon.ico' />
       </Head>
 
-      <TelnyxMeetContext.Provider
-        value={{
-          audioInputDeviceId,
-          audioOutputDeviceId,
-          videoInputDeviceId,
-          setAudioInputDeviceId,
-          setAudioOutputDeviceId,
-          setVideoInputDeviceId,
-          isAudioTrackEnabled,
-          isVideoTrackEnabled,
-          setIsAudioTrackEnabled,
-          setIsVideoTrackEnabled,
-          sendNotification,
-          networkMetrics,
-          setNetworkMetrics,
-          unreadMessages,
-          optionalFeatures,
-        }}
-      >
-        <Main align='center' justify='center' background='light-2'>
-          <Toaster />
-
+      <Main align='center' justify='center' background='light-2'>
+        <Toaster />
+
+        <TelnyxMeetContext.Provider
```

Comment:
Moving `main` and `Toaster` outside the context provider, is a personal preference, since these components do not require this context at any point.